packages/client3/components/dashboard/my-open-pool/my-open-pool.tsx

```diff
@@ -0,0 +1,21 @@
+import { BodyText, Button } from "../../design-system";
+
+type Props = {
+  isVerified: boolean;
+  onButtonClick: () => void;
+};
+
+export function MyOpenPool({ isVerified, onButtonClick }: Props) {
```

Comment:
Not sure how I feel about "MyOpenPool"

I would suggest changing it to `ArtristPool` or something.